src/app/clusters/scenes-server/scenes-server.cpp

```diff
@@ -115,22 +115,34 @@ CHIP_ERROR ScenesServer::Init()
 
     for (auto endpoint : EnabledEndpointsWithServerCluster(Id))
     {
-        // Explicit AttributeValuePairs is mandatory for matter so we force it here, ScenesName is not but it is forced for now
-        // TODO: We currently force SceneNames on but this needs to be modified to read the value generated from Zap instead.
-        uint32_t featureMap  = to_underlying(Feature::kExplicit) | to_underlying(Feature::kSceneNames);
-        EmberAfStatus status = Attributes::FeatureMap::Set(endpoint, featureMap);
+        uint32_t featureMap;
```

Comment:
Pre-existing, but this logic is wrong, because it will fail for dynamically added endpoints, no?  This logic should live in the per-endpoint init, not global init....  Followup for that, please.